### PR TITLE
Support for deploying other dropwizard commands and disabling of autoregistration in consul

### PR DESCRIPTION
**Pull request by pchojnacki**

### CODE REVIEW

File: service/indexer-base/build.gradle [hunk 1]
```diff
@@ -1,32 +1,10 @@
-version = "0.1.9-SNAPSHOT"
-apply from: rootProject.file('gradle/dropwizard_service.gradle')
-mainClassName = "com.wikia.services.indexerbase.IndexerBaseApplication"
+version = "0.0.1-SNAPSHOT"
```
Comment (sqreek) [inline service/indexer-base/build.gradle]: This is lower version that was earlier.
Why?

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/IndexerBaseApplication.java [hunk 2]
```diff
@@ -33,6 +42,11 @@ public void run(IndexerConfiguration configuration, Environment environment)
   }
 
   @Override
+  public List<Class<? extends FeatureInstaller>> getDisabledFeatureInstallers() {
+    return Collections.singletonList(ConsulRegistrationInstaller.class);
```
Comment (sqreek) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/IndexerBaseApplication.java]: Is it not handle by [enableConsulAutoregistration](https://github.com/Wikia/pandora/pull/275/files#diff-332ade6843774193d486b29c264d37a5R105)

File: lib/dropwizard/consul/src/main/java/com/wikia/dropwizard/consul/ConsulGuiceApplication.java [hunk 3]
```diff
@@ -31,6 +30,16 @@
  */
 public class ConsulGuiceApplication<T extends Configuration> extends Application<T> {
 
+  protected Boolean enableConsulAutoregistration;
```
Comment (sqreek) [inline lib/dropwizard/consul/src/main/java/com/wikia/dropwizard/consul/ConsulGuiceApplication.java]: Comment at this will be helpful

**Comment by sqreek** [comment]
There is still failing tests from nirvana code generation.
AFAIK @nmonterroso fix it
except questions, description should be provided.
otherwise LGTM

File: service/indexer-base/build.gradle (hunk 1 above)
Comment (pchojnacki) [inline service/indexer-base/build.gradle]: applications shouldn't start their versionning at 0.1.9-SNAPSHOT

File: service/indexer-base/src/main/java/com/wikia/services/indexerbase/IndexerBaseApplication.java (hunk 2 above)
Comment (pchojnacki) [inline service/indexer-base/src/main/java/com/wikia/services/indexerbase/IndexerBaseApplication.java]: I'll remove it, its just junk code

**Comment by sqreek** [comment]
this [merge](https://github.com/Wikia/pandora/pull/276) fix tests.

**Pull request merged**